react/w-books/src/app/components/Button/index.tsx

```diff
@@ -10,11 +11,28 @@ interface ButtonProps {
   isSubmit?: boolean;
   isWidthAuto?: boolean;
   handleClick?: () => void;
+  isLink?: boolean;
+  to?: string;
 }
 
-export default function Button({ children, isSubmit, className, isFilled, isWidthAuto, handleClick }: ButtonProps) {
+export default function Button({ children, isSubmit, className, isFilled, isWidthAuto, handleClick, to, isLink }: ButtonProps) {
 
   return (
+    isLink ?
+    <Link
+      to={to ?? '/'}
+      type={isSubmit ? 'submit' : 'button'}
```

Comment:
the goal of this component is to avoid duplicating the scss module, they are both visually the same. Do I have to duplicate styles in two files or can I share the scss module?

Reply:
ahh perfect, understand